Index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Zoo Dekadenzia</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="CSS/Style.css">
+    <meta name="viewport" content="width=device-width, initial-scale1">
+    
+
+    <meta name="description" content="HTML">
+</head>
+
+<body>
+    <header>
+        <div class="customheader background-baer">
+        <h1 class="h1header">Willkommen</h1>
+    </div>
+
+
+
+        <nav class="Navigation">
+            <ul>
+                <strong> <li> <a href="Index.html">Willkommen!</a></li> </strong>
```

Comment:
besser wäre:
Style:
nav li a { font-weight: bold; }
wenn im html dann:
li> strong> a href....>
strong ist ein inline element und gehört in den tag li